llvm/include/llvm/IR/DebugProgramInstruction.h

```diff
@@ -157,6 +157,11 @@ class DbgRecord : public ilist_node<DbgRecord> {
   ~DbgRecord() = default;
 };
 
+inline raw_ostream &operator<<(raw_ostream &OS, const DbgRecord &R) {
+  R.print(OS);
+  return OS;
+}
+
```

Comment:
One more thing, since `DbgRecord::print` already dispatches, we could simply replace the `DPValue` operator with this, right?